src/overlays/actors/ovl_En_Brob/z_en_brob.h

```diff
@@ -6,9 +6,19 @@
 
 struct EnBrob;
 
+typedef void (*EnBrobUpdateFunc)(struct EnBrob* this, GlobalContext* globalCtx);
```

Comment:
This should be `ActionFunc`. `UpdateFunc` is used when the function is set in `Init` and stays fixed.